src/riak_cs_riak_client.erl

```diff
@@ -113,6 +114,36 @@ pbc_pool_name(undefined) ->
 pbc_pool_name(BagId) when is_binary(BagId) ->
     list_to_atom(lists:flatten(io_lib:format("pbc_pool_~s", [BagId]))).
 
+%% @doc Make a thunk that looks up samples for a given bucket+prefix.
+-spec rts_puller(riak_client(), binary(), iolist(), riak_cs_stats:key()) -> fun().
+rts_puller(RcPid, Bucket, Postfix, StatsKey) ->
```

Comment:
Looks like `Suffix` is major (Japanese page => http://blog.livedoor.jp/lalha/archives/50165797.html )